XpringKit/Org_Xrpl_Rpc_V1_CurrencyAmount+XRPCurrency.swift

```diff
@@ -0,0 +1,6 @@
+internal extension XRPCurrency {
```

Comment:
What is an `internal extension` in Swiftland?

Reply:
Extension is adding functionality to this class. 

This is adding a new initializer to XRPCurrency that takes a protocol buffer. We could do this in one file but it's idiomatic in Swift to separate out into extensions when you're tightly coupling two classes.